pkgs/development/tools/geckodriver/default.nix

```diff
@@ -1,26 +1,30 @@
 { lib
-, fetchFromGitHub
+, fetchzip
 , rustPlatform
 , stdenv
 , darwin
 }:
 
-with rustPlatform; 
-
-buildRustPackage rec {
-  version = "0.22.0";
+rustPlatform.buildRustPackage {
+  version = "0.26.0";
   pname = "geckodriver";
+  sourceRoot = "source/testing/geckodriver";
 
-  src = fetchFromGitHub {
-    owner = "mozilla";
-    repo = "geckodriver";
-    rev = "v${version}";
-    sha256 = "12m95lfqwdxs2m5kjh3yrpm9w2li5m8n3fw46a2nkxyfw6c94l4b";
-  };
+  src = let
+    # Source revisions are noted alongside the binary releases:
+    # https://github.com/mozilla/geckodriver/releases
+    rev = "e9783a644016aa9b317887076618425586730d73";
```

Comment:
I think it would be cleaner to just inline this.

Reply:
Somewhat strongly disagree, but can do.

Reply:
If you strongly disagree, feel free to leave it as you like. :)

Reply:
*shrug* I'm not invested either way :p